.github/workflows/run_scrapers.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         pip install -r requirements.txt
         sudo apt update || true # do not fail if update does not work
         sudo apt-get install sqlite3
-        if [ "$SCRAPER_KEY" = "FL" ] || [ "$SCRAPER_KEY" = "GE" ] || [ "$SCRAPER_KEY" = "VD" ] || [ "$SCRAPER_KEY" = "GL" ] || [ "$SCRAPER_KEY" = "SZ" ] || [ "$SCRAPER_KEY" = "TI" ] || [ "$SCRAPER_KEY" = "SO" ] ; then
+        if [ "$SCRAPER_KEY" = "FL" ] || [ "$SCRAPER_KEY" = "GE" ] || [ "$SCRAPER_KEY" = "VD" ] || [ "$SCRAPER_KEY" = "GL" ] || [ "$SCRAPER_KEY" = "SZ" ] || [ "$SCRAPER_KEY" = "TI" ] || [ "$SCRAPER_KEY" = "SO" ] || [ "$SCRAPER_KEY" = "VS" ] ; then
```

Comment:
It's probay time to get rid of this condition andmsimply install it for all cantons.

It made sense when only a couple of cantons needed PDF scraping capabilities.

Reply:
I thought about that as well, should be addressed in 77f43e3.
Or should we just have one `sudo apt-get install xxx` call with sqlite3 and poppler-utils in one shot?